src/v2i-hub/PortDrayagePlugin/src/PortDrayagePlugin.cpp

```diff
@@ -30,17 +31,37 @@ PortDrayagePlugin::~PortDrayagePlugin() {
 }
 
 void PortDrayagePlugin::UpdateConfigSettings() {
-	// Update configuration
+
 	lock_guard<mutex> lock(_cfgLock);
+
+	// Database configuration
 	GetConfigValue<string>("Database_Username", _database_username);
 	GetConfigValue<string>("Database_Password", _database_password);
 	GetConfigValue<string>("Database_IP",_database_ip);
 	GetConfigValue<uint16_t>("Database_Port",_database_port);
 	GetConfigValue<string>("Database_Name", _database_name);
-	std::string loglevel;
-	GetConfigValue<string>("LogLevel", loglevel);
 
-	FILELog::ReportingLevel() = FILELog::FromString(loglevel);
+	// Port Drayage Web Service Configuration
+	uint16_t polling_frequency;
```

Comment:
Initialize variables to default values.

Reply:
They are initialized to default. If no config value is present it takes the default value